### PR TITLE
[#268 ] RABC authentication feature 

### PR DESCRIPTION
**Pull request by GuoYL123**

### CODE REVIEW

File: spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/registry/ServiceCombWatcher.java [hunk 1]
```diff
@@ -138,6 +142,9 @@ public void checkServerTrusted(X509Certificate[] chain,
   private WebSocketClient buildClient() {
     Map<String, String> signedHeader = new HashMap<>();
     signedHeader.put("x-domain-name", ServiceRegistryConfig.DEFAULT_PROJECT);
+    if (TokenCache.getToken() != null) {
+      signedHeader.put("Authorization", "Bearer " + TokenCache.getToken().getToken());
```
Comment (tianxiaoliang) [inline spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/registry/ServiceCombWatcher.java:146]: TokenCache.getToken().getToken()写法不太好，概念重叠

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DefaultHttpTransport.java [hunk 2]
```diff
@@ -51,12 +62,59 @@
...
+  private void getToken(ServiceCombRBACProperties serviceCombRBACProperties, List<String> urlList) {
+    Response response = null;
+    String url;
+    for (String s : urlList) {
+      url = s;
+      try {
+        StringEntity stringEntity = new StringEntity(
+            JsonUtils.OBJ_MAPPER.writeValueAsString(serviceCombRBACProperties), "utf-8");
+        response = this.sendPostRequest(url + "/v4/token", stringEntity);
+        if (response.getStatusCode() == 200) {
```
Comment (tianxiaoliang) [inline spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DefaultHttpTransport.java:92]: 200 能否用常量

**Review by tianxiaoliang — CHANGES_REQUESTED** [review]

File: spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/registry/ServiceCombWatcher.java (hunk 1 above)
Comment (GuoYL123) [reply spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/registry/ServiceCombWatcher.java]: done

File: spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DefaultHttpTransport.java (hunk 2 above)
Comment (GuoYL123) [reply spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DefaultHttpTransport.java]: done

**Pull request merged**